includes/polling/applications/rrdcached.inc.php

```diff
@@ -43,8 +43,12 @@
     if (!$sock) {
         $socket = \LibreNMS\Config::get('rrdcached');
         if (substr($socket, 0, 6) == 'unix:/') {
-            if (file_exists(substr($socket, 5))) {
-                $sock = fsockopen($socket);
+            $socket_file = substr($socket, 5);
+            if (file_exists($socket_file)) {
+                $sock = fsockopen("unix://" . $socket_file);
+                if (!$sock) {
+                    $sock = fsockopen("unix:" . $socket_file);
```

Comment:
why do you expect one of these to fail?

Reply:
i personally know only the unix://<fullpathtofile> way. But after discussion on friday i'm a little bit confused if unix:<fullpathtofile> also somewhere is valid, but don't think so,
If code would be only for myself i wouldn't implement the "if not sock"-case.
Can remove it, it's no problem (would be very fine with it ;) )

Reply:
it is because rrdtool and fsockopen accept different formats.

Reply:
done :-)